Dockerfile

```diff
@@ -1,20 +1,26 @@
 ARG DOCKER_PLATFORM
 ARG DOCKER_ARCH
 
-FROM --platform=${DOCKER_PLATFORM:-linux/amd64} ${DOCKER_ARCH:-amd64}/debian:buster-slim
+FROM --platform=${DOCKER_PLATFORM:-linux/amd64} ${DOCKER_ARCH:-amd64}/debian:bookworm-slim
 ARG GOARCH
 ARG VERSION_SERVER_URL
 ARG SUPERVISOR_SERVER
-ENV GOARCH ${GOARCH:-amd64}
-ENV VERSION_SERVER_URL ${VERSION_SERVER_URL:-https://version.storj.io}
-ENV SUPERVISOR_SERVER ${SUPERVISOR_SERVER:-unix}
+ENV GOARCH=${GOARCH:-amd64}
```

Comment:
Good change about updating the syntax.

Because the new syntax allows defining multiple env vars in the same instruction, I suggest setting all of them in single instructions rather than spread into three.